{"name": "test_login_user[17647352440-yy123456-False-200002-用户名或密码错误]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200002 == '200002'", "trace": "self = <testcases.api_test.test_01_login_erp.TestUserLogin object at 0x0000025848BA64F0>\nbase_url = 'https://erp-qa.qdjk.xyz', username = '17647352440'\npassword = 'yy123456', except_result = False, except_code = '200002'\nexcept_msg = '用户名或密码错误'\n\n    @allure.story(\"用例--登录用户\")\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             login_data[\"test_login_user\"])\n    def test_login_user(self,base_url, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = login_erp_user(base_url,username, password)\n        assert result.success == except_result, result.error\n        assert result.success == except_result, result.error\n        logger.info(\"code ==>> 期望结果：{}， 实际结果：【 {} 】\".format(except_code, result.response.get(\"code\")))\n        print(result.response)\n>       assert result.status_code == except_code\nE       AssertionError: assert 200002 == '200002'\n\ntest_01_login_erp.py:23: AssertionError"}, "attachments": [{"name": "log", "source": "9ce6a330-f4f8-4d78-8c0f-61e799fdc643-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "1d4848e2-1389-4802-9639-01036ee6512a-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0ae76dd5-0f73-435c-9691-4fe6831d363c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'17647352440'"}, {"name": "password", "value": "'yy123456'"}, {"name": "except_result", "value": "False"}, {"name": "except_code", "value": "'200002'"}, {"name": "except_msg", "value": "'用户名或密码错误'"}], "start": 1694413333329, "stop": 1694413334029, "uuid": "39b5b6a0-f270-4c4c-822d-09e3cd9f2de7", "historyId": "c34e0cd68573d7a4c0dd7563c1b60da5", "testCaseId": "80c254abc9876d28cea0e8148b92d53d", "fullName": "testcases.api_test.test_01_login_erp.TestUserLogin#test_login_user", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "用例--登录用户"}, {"name": "feature", "value": "用户登录模块"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_01_login_erp"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "WIN-FOCJ805QQO5"}, {"name": "thread", "value": "15500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_01_login_erp"}]}